processor/elasticinframetricsprocessor/processor.go

```diff
@@ -57,16 +58,38 @@ func newProcessor(set processor.Settings, cfg *Config) *ElasticinframetricsProce
 
 // processMetrics processes the given metrics and applies remappers if configured.
 func (p *ElasticinframetricsProcessor) processMetrics(_ context.Context, md pmetric.Metrics) (pmetric.Metrics, error) {
+	newmetic := pmetric.NewMetrics()
+	rmnew := newmetic.ResourceMetrics().AppendEmpty()
+	rmscope := rmnew.ScopeMetrics().AppendEmpty()
 	for i := 0; i < md.ResourceMetrics().Len(); i++ {
 		resourceMetric := md.ResourceMetrics().At(i)
 		rm := resourceMetric.Resource()
-
 		for j := 0; j < resourceMetric.ScopeMetrics().Len(); j++ {
 			scopeMetric := resourceMetric.ScopeMetrics().At(j)
 			for _, r := range p.remappers {
 				r.Remap(scopeMetric, scopeMetric.Metrics(), rm)
 			}
 		}
 	}
+
+	if p.cfg.Override {
+		for i := 0; i < md.ResourceMetrics().Len(); i++ {
+			resourceMetric := md.ResourceMetrics().At(i)
+
+			for j := 0; j < resourceMetric.ScopeMetrics().Len(); j++ {
+				scopeMetric := resourceMetric.ScopeMetrics().At(j)
+				for l := 0; l < scopeMetric.Metrics().Len(); l++ {
+					metric := scopeMetric.Metrics().At(l)
+					p.logger.Info("See", zap.String("NAME:", metric.Name()), zap.Int("I:", i), zap.Int("J:", j), zap.Int("L:", l), zap.String("Length:", scopeMetric.Scope().Name()))
+					if strings.HasPrefix(metric.Name(), "kubernetes.pod") {
```

Comment:
I think the "override" feature should not rely on specific metrics string prefixes like "kubernetes.pod", but be agnostic to the underlying remapped metrics. By "override", we should expect the processor to only output the remapped metrics, the input ones should be discarded. In that sense, the processor will only output "ECS" metrics.

Reply:
Agreed ! So I changed the implementation based on [otel_remapped label](https://github.com/elastic/opentelemetry-lib/blob/6d89cbad4221429570107eb4a4968cf8a2ff919f/remappers/common/const.go#L31)